src/test/kotlin/com/chloe/kotlinserv/TestGetCountryStats.kt

```diff
@@ -0,0 +1,288 @@
+package com.chloe.kotlinserv
+
+import com.chloe.kotlinserv.http.HttpRequest
+import com.chloe.kotlinserv.http.HttpResponse
+import com.chloe.kotlinserv.model.CountryStats
+import com.chloe.kotlinserv.route.GetCountryStatsRoute
+import com.chloe.kotlinserv.service.GeoDataServiceImpl
+import io.mockk.confirmVerified
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.junit.jupiter.api.Test
+import org.skyscreamer.jsonassert.JSONAssert
+import org.skyscreamer.jsonassert.JSONCompareMode
+import kotlin.test.assertEquals
+
+class TestGetCountryStats {
+    @Test
+    fun testProcessFunction_WhenServiceRespondWithGroupedLocalData_ShouldReturn200CodeWithBody() {
+        val serviceImpl = mockk<GeoDataServiceImpl>()
```

Comment:
serviceImpl and getCountryStatsRoute could be extracted to class fields and initialized there